schemas/context/normalized-experianceevent.json

```diff
@@ -0,0 +1,135 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/normalized-experienceevent",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Normalized ExperienceEvent Mixin",
+  "type": "object",
+  "description": "",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/context/experienceevent-advertising",
+    "https://ns.adobe.com/xdm/context/experienceevent-application",
+    "https://ns.adobe.com/xdm/context/experienceevent-channel",
+    "https://ns.adobe.com/xdm/context/experienceevent-commerce",
+    "https://ns.adobe.com/xdm/context/experienceevent-directmarketing",
+    "https://ns.adobe.com/xdm/context/experienceevent-environment-details",
+    "https://ns.adobe.com/xdm/context/experienceevent-marketing",
+    "https://ns.adobe.com/xdm/context/experienceevent-media",
+    "https://ns.adobe.com/xdm/context/experienceevent-profile-stitch",
+    "https://ns.adobe.com/xdm/context/experienceevent-search",
+    "https://ns.adobe.com/xdm/context/experienceevent-segmentmembership",
+    "https://ns.adobe.com/xdm/context/experienceevent-technical-details",
+    "https://ns.adobe.com/xdm/context/experienceevent-web"],
+  "definitions": {
+    "normalized-experienceevent": {
+      "properties": {
+          "xdm:eventType": {
```

Comment:
A context/experienceevent (which this mixin is meta:intendedToExtend) already include the data/time-series schema which has an eventType field already.